src/cmds/qsub.c

```diff
@@ -5773,15 +5782,15 @@ do_daemon_stuff(void)
 	}
 
 out:
-	close(bindfd);
 	unlink(fl);
+	close(bindfd);
```

Comment:
Why swap the order of the unlink() and the close()?   The man page says if you unlink a socket, any process that still has it open can continue to use it.  So this can happen in either order and it's fine.  I'd just think it'd make more sense to close something before you unlink it.

Reply:
yes, in this place unlink can come after close